// react-datocms/dist/Seo/__tests__/index.test.js
"use strict";
var __importStar = (this && this.__importStar) || function (mod) {
    if (mod && mod.__esModule) return mod;
    var result = {};
    if (mod != null) for (var k in mod) if (Object.hasOwnProperty.call(mod, k)) result[k] = mod[k];
    result["default"] = mod;
    return result;
};
exports.__esModule = true;
var React = __importStar(require("react"));
var enzyme_1 = require("enzyme");
var index_1 = require("../index");
var metaTags = [
    {
        attributes: null,
        content: "A new Media Area is online! - DatoCMS",
        tag: "title"
    },
    {
        attributes: {
            property: "og:title",
            content: "A new Media Area is online!"
        },
        content: null,
        tag: "meta"
    },
    {
        attributes: {
            name: "twitter:title",
            content: "A new Media Area is online!"
        },
        content: null,
        tag: "meta"
    }
];
describe("renderMetaTags", function () {
    it("generates an array of meta tags", function () {
        var wrapper = enzyme_1.shallow(React.createElement("head", null, index_1.renderMetaTags(metaTags)));
        expect(wrapper).toMatchSnapshot();
    });
});
describe("renderMetaTagsToString", function () {
    it("generates an array of meta tags", function () {
        var result = index_1.renderMetaTagsToString(metaTags);
        expect(result).toEqual([
            '<title>A new Media Area is online! - DatoCMS</title>',
            '<meta property="og:title" content="A new Media Area is online!" />',
            '<meta name="twitter:title" content="A new Media Area is online!" />',
        ].join("\n"));
    });
});
